src/scs/pki.md

```diff
@@ -37,3 +37,9 @@ In a two-tier hierarchy the Root CA and issuing (Subordinate) CAs are separated
 
 In a three-tier CA an intermediate CA is placed between the Root CA and the Subordinate (issuing) CA. This is done to further separate the Root CA from low-level CA operations. The middle layer (intermediate CA) is only used to sign Subordinate CAs that issue the end-entity certificates. 
 
+## Proposed CA Hierarchy
+
+The following diagram describes proposed CA for the SCS. The three-tier CA is chosen based on high security level and potential need to scale it to several projects later on, keeping the main control under the same RootCA.
```

Comment:
The following diagram describes **the** proposed CA for the SCS. The three-tier CA is chosen based on **the** **high-security** level and **the** potential need to scale it to several projects **,** later on, keeping the main control under the same Root CA.